OpenOversight/app/utils.py

```diff
@@ -609,10 +607,17 @@ def upload_image_to_s3_and_store_in_db(image_buf, user_id, department_id=None):
     we have to scrub the image before we do anything else like hash it
     but we also have to get the date for the image before we scrub it.
     """
-    image_buf.seek(0)
-    image_type = imghdr.what(image_buf)
+    kind = image_match(image_buf)
+    image_type = kind.extension if kind else None
```

Comment:
Should the `.lower()` on the extension be done here? That way the check against allowed extensions is performed with a normalized extension name?

Reply:
Good question! This `extension` is returned by matchers from the filetype.py library which are generally lowercase already but I can definitely change it here. Realizing that I also did that later on in the code so I should keep it consistent.

What do you think?

Reply:
Just to be safe, let's move it up here!

Reply:
Done!